client/src/components/Navbar/Navbar.css

```diff
@@ -1,34 +1,33 @@
-/* Navbar.css */
-.navbar {
-  background-color: rgb(4, 16, 36); /* Set your desired background color */
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  padding: 10px 20px;
-}
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
 
-.logo a {
-  color: #fff; /* Set the text color for the logo */
-  text-decoration: none;
-  font-size: 24px; /* Adjust the font size as needed */
-  font-weight: bold;
+/* Custom keyframe animation for fadeIn */
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
 }
 
-.nav-list {
-  list-style: none;
-  display: flex;
-}
+/* Custom styles */
+.button {
+  color: aqua;
+  background-color: teal; /* Set your desired background color */
+  padding: 0.5rem 1rem; /* Adjust padding as needed */
+  border-radius: 0.25rem; /* Add border-radius for rounded corners */
 
-.nav-list li {
-  margin-right: 20px; /* Adjust the spacing between list items */
-}
+  /* Add a transition for smoother hover effect */
+  transition: background-color 0.3s ease;
 
-.nav-list li:last-child {
-  margin-right: 0; /* Remove margin from the last item to prevent extra spacing */
-}
+  /* Optional: Add an animation for extra visual appeal */
+  animation: fadeIn 1s ease;
 
```

Comment:
Looks good to me but I need a `div` when I hover through the  **Learn Lesson** button